test/powershell/Modules/PSDiagnostics/PSDiagnostics.Tests.ps1

```diff
@@ -0,0 +1,93 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+Describe "PSDiagnostics cmdlets tests." -Tag "CI", "RequireAdminOnWindows" {
+    BeforeAll {
+        $LogType = 'Analytic'
+        $OriginalDefaultParameterValues = $PSDefaultParameterValues.Clone()
+        if (-not $IsWindows) {
+            $PSDefaultParameterValues["it:skip"] = $true
+        }
+        else{
+            $LogSettingBak = Get-LogProperties -Name Microsoft-Windows-PowerShell/$LogType
+        }
+    }
+    AfterAll {
+        if ($IsWindows) {
+            Set-LogProperties -LogDetails $LogSettingBak -Force
+        }
+        $Global:PSDefaultParameterValues = $OriginalDefaultParameterValues
+    }
+
+    Context "Test for Enable-PSTrace and Disable-PSTrace cmdlets." {
+        it "Should enable $LogType logs for Microsoft-Windows-PowerShell." {
+            [XML]$CurrentSetting = & wevtutil gl Microsoft-Windows-PowerShell/$LogType /f:xml
+            if($CurrentSetting.Channel.Enabled -eq 'true'){
+                & wevtutil sl Microsoft-Windows-PowerShell/$LogType /e:false /q
+            }
+
+            Enable-PSTrace -Force
+
+            [XML]$ExpectedOutput = & wevtutil gl Microsoft-Windows-PowerShell/$LogType /f:xml
+
+            $ExpectedOutput.Channel.enabled | Should -BeExactly 'true'
+        }
+
+        it "Should disable $LogType logs for Microsoft-Windows-PowerShell." {
+            [XML]$CurrentState = & wevtutil gl Microsoft-Windows-PowerShell/$LogType /f:xml
+            if($CurrentState.channel.enabled -eq 'false'){
+                & wevtutil sl Microsoft-Windows-PowerShell/Analytic /e:true /q
```

Comment:
Should this be `Microsoft-Windows-PowerShell/Analytic` -> `Microsoft-Windows-PowerShell/$LogType`

Reply:
oops, yes, that was a copy paster from test code.